.github/workflows/ci_static-checks.yaml

```diff
@@ -67,6 +67,10 @@ jobs:
                         ${{ steps.composer-cache.outputs.dir }}
                     key: ${{ github.run_id }}-${{ runner.os }}-${{ hashFiles('composer.json') }}-symfony-${{ matrix.symfony }}-api-platform-${{ matrix.api-platform }}
                     
+            -   name: "Require ext-random"
+                if: matrix.php == '8.2'
+                run: composer require ext-random --no-update --no-scripts --no-interaction
+                    
```

Comment:
This extension exists only in `8.2` and is required to be present to satisfy ComposerRequireCheck.